test/Directory.Build.props

```diff
@@ -31,6 +31,20 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="2.4.0-prerelease-63213-02" />
+    <PackageReference Include="coverlet.msbuild" Version="$(CoverletVersion)" />
+    <PackageReference Include="Codecov" Version="$(CodecovVersion)" />
```

Comment:
You don't want this PackageReference here as that means that every test assemblies references Codecov which probably doesn't even make sense. Make sure that you restore that tool as part of the repo bootstrapping.

Reply:
@safern Does adding reference to codecov in codecov.proj will load the tool before the execution?

Reply:
I think you would just need to add it into: https://github.com/dotnet/machinelearning/blob/master/init-tools.msbuild#L9 since that is the project that is used to bootstrap external tools for the repo. 

Or you could just make sure that your Codecov target depends on Restore: https://github.com/dotnet/machinelearning/pull/2194/files#diff-180e04fde140ade031905faef1939141R5

And add the package references to that project itself.

Reply:
Thanks!